asserts/asserts_test.go

```diff
@@ -56,34 +59,36 @@ func (as *assertsSuite) TestDecodeEmptyBodyAllDefaults(c *C) {
 	c.Check(a.AuthorityID(), Equals, "auth-id1")
 }
 
+const exampleEmptyBody2NlNl = "type: test-only\n" +
+	"authority-id: auth-id1\n" +
+	"revision: 0\n" +
+	"body-length: 0" +
+	"\n\n" +
+	"\n\n" +
+	"openpgp c2ln\n"
```

Comment:
but that's a problem of the signatures, \n is only interesting because it can be confused/part of separators,  the only strong rule we have on signatures is that they cannot contain "\n\n"